python/test_notebooks/notebook_tests.py

```diff
@@ -13,6 +13,7 @@
     "Writing_Reference_Profiles_to_WhyLabs.ipynb",
     "flask_with_whylogs.ipynb",
     "BigQuery_Example.ipynb",
+    "Segments.ipynb",
```

Comment:
I agree with @jamie256  that I don't think we should test this due to the user input.